cmd/oci-create-runtime-bundle/oci-create-runtime-bundle.1.md

```diff
@@ -24,7 +24,7 @@ runtime-spec-compatible `dest/config.json`.
   A directory representing the root filesystem of the container in the OCI runtime bundle. It is strongly recommended to keep the default value. (default "rootfs")
 
 **--type**
-  Type of the file to unpack. If unset, oci-create-runtime-bundle will try to auto-detect the type. One of "imageLayout,image"
+  Type of the file to create runtime bundle. If unset, oci-create-runtime-bundle will try to auto-detect the type. One of "imageLayout,image"
```

Comment:
Yes, maybe it is redundant here to cover above description. And I'd prefer to old words than "source type".